charts/ingress-nginx/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: ingress-nginx
 # When the version is modified, make sure the artifacthub.io/changes list is updated
 # Also update CHANGELOG.md
-version: 4.0.0
-appVersion: 1.0.0-beta.1
+version: 4.0.0-beta1
```

Comment:
The current release in development is ;
```
% helm search repo ingress --devel
NAME                            CHART VERSION   APP VERSION     DESCRIPTION                                                                                                                                
ingress-nginx/ingress-nginx     4.0.0-beta.1    1.0.0-beta.1    Ingress controller for Kubernetes using NGINX a...
```
So maybe ;
```
version: 4.0.0-beta.2
```

Reply:
yeeah you are right. Fixing!